lib/fastlane/plugin/menigit/actions/ensure_no_changes.rb

```diff
@@ -34,7 +35,12 @@ def self.available_options
                                        description: "The flag whether to show the git diff if the repo is dirty",
                                        optional: true,
                                        default_value: false,
-                                       type: Boolean)
+                                       type: Boolean),
+          FastlaneCore::ConfigItem.new(key: :error_message,
+                                        env_name: "ENSURE_NO_CHANGES_ERROR_MESSAGE",
+                                        description: "Error message that will be show instead of default one based on path",
```

Comment:
Isn't it better to just make the message more clear instead of allowing the user to set one?